pkg/services/org/orgimpl/store.go

```diff
@@ -76,9 +76,12 @@ func (ss *sqlStore) Insert(ctx context.Context, org *org.Org) (int64, error) {
 	var orgID int64
 	var err error
 	err = ss.db.WithDbSession(ctx, func(sess *db.Session) error {
-		if orgID, err = sess.InsertOne(org); err != nil {
+		if _, err = sess.Insert(org); err != nil {
 			return err
 		}
+
+		orgID = org.ID
```

Comment:
do we want to have checks here instead of the org.ID != 0 down below?

Reply:
trying to minimize flow changes for this fix due to backport and undefined behavior.

For 9.5 we can start looking at shoring up the tests for this and adding more guards